src/store/note_record/mod.rs

```
@@ -14,6 +14,7 @@ mod output_note_record;
 pub use input_note_record::InputNoteRecord;
 pub use output_note_record::OutputNoteRecord;
 
+#[allow(rustdoc::broken_intra_doc_links)]
```

Because `[SqliteStore](crate::store::sqlite_store::SqliteStore)` is compiled out with the "wasm" feature, this code fails the `cargo make doc` command in the github actions workflows. Added this to suppress the error. Happy to change this is a different route is preferred!

---

Could we make it something like this?

`#![cfg_attr(feature = "wasm", allow(rustdoc::broken_intra_doc_links))]`

---

Should we do it like this?

```rust
#![cfg_attr(feature = "wasm", allow(rustdoc::broken_intra_doc_links))]
```

---

Yep, this is much better. Thank you!